packages/core/src/progress-circle/progress-circle.element.spec.ts

```diff
@@ -155,4 +155,160 @@ describe('progress circle element – ', () => {
     });
   });
   // circle classname arcstroke/backstroke
+
+  describe('ariaLabelTemplate', () => {
+    it('should return expected default aria label template', async () => {
+      await componentIsStable(component);
+      expect(component.getAttribute('aria-label')).toBe('Loading 49%');
+    });
+    it('should allow for custom aria label template', async () => {
+      await componentIsStable(component);
+      component.setAttribute('aria-label-template', 'Now $i18n... $val out of 100 items');
```

Comment:
I don't clearly understand the idea about the template. Isn't it enough that they can override the label with their own custom string? Why do we want to handle these dynamic updates internally?

Reply:
this feature has been removed in favor of a more general implementation